src/main/java/fi/otavanopisto/kuntaapi/server/controllers/ClusterController.java

```diff
@@ -0,0 +1,65 @@
+package fi.otavanopisto.kuntaapi.server.controllers;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.PostConstruct;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import org.wildfly.clustering.group.Group;
+import org.wildfly.clustering.group.Node;
+
+@ApplicationScoped
+public class ClusterController {
+
+  @Inject
+  private Logger logger;
+  
+  private Group channelGroup;
+  
+  @PostConstruct
+  private void postConstruct() {
+    try {  
+      channelGroup = (Group) new InitialContext().lookup("java:jboss/clustering/group/web");
+    } catch (NamingException ex) {
+      logger.log(Level.SEVERE, "Cannot find java:jboss/clustering/group/web disabling task distribution");
+    }
+  }
+
+  public String getLocalNodeName() {
+    if (channelGroup == null) {
+      return "UNKNOWN";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "UNKNOWN" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)